test/erc721/index.js

```diff
@@ -1149,6 +1167,45 @@ module.exports = (init, extensions) => {
     });
   }
 
+  if (extensions.burnAsset) {
```

Comment:
This is acopy/paste of `if (extensions.burn) {...` but using the signature of the `burn` function from `ERC1155ERC721.sol`